plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/CleanupOldDeploysJobTest.java

```diff
@@ -62,17 +63,17 @@ private void testRun(int directoryCount, String[] expectedDirectoriesToKeep) thr
 
     IPath tempFolderPath = new Path(tempFolder.getRoot().toString());
     CleanupOldDeploysJob job = new CleanupOldDeploysJob(tempFolderPath);
-    job.run(mock(IProgressMonitor.class));
+    assertEquals(Status.OK_STATUS, job.run(new NullProgressMonitor()));
```

Comment:
You should be able to use `null` for the progress monitor (though this is perfectly acceptable)